### DISCUSSION TITLE
2023-11-23 CUE Modules Feedback Session / office hours

### DISCUSSION THREAD

**Discussion by dominikdm** [cue-lang/cue#2700] [Community Calls]
This discussion is a placeholder for the next instalment in the Modules Feedback Sessions (office hours).

See this discussion for context on the modules proposal: https://github.com/cue-lang/cue/discussions/2330

The next session will be on **Thursday, 23 Nov, 2023 at 1600 UTC.**

Please reply with any questions/topics you would like to see covered.

### Rough agenda
Will be updated shortly. 

### Meeting details
https://meet.google.com/wis-hrtw-isf

This session will be recorded for those unable to join us live.

### CUE Community shared calendar
[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).

### Recording
Will be added after the call. 

### Slides/materials
Will be made available after the call. 

### Auto-generated transcript
Will be added after the call.

**Comment by verdverm** [comment]
FYI, that is a holiday in the US

**Reply by dominikdm** [reply]
Thanks! Yes, we know this will be a special modules session - the Thanksgiving edition! By recording the calls, and having another session in just two weeks (already in the shared calendar, announcement follows after this one to avoid confusion), we believe we can ensure nobody misses anything, independent of holiday schedules. If you have any questions upfront or topics you want to see covered, please let us know.

**Comment by myitcv** [comment]
On reflection we are going to cancel tomorrow's call. Roger and Daniel have been preparing for and attending GoLab (videos to follow!), and so we can roll updates into the next call. This also gives folks in the US a chance to attend in person. @dominikdm will schedule in the next couple of days.
